ContextApp/src/main/java/com/github/migi_1/ContextApp/MainActivity.java

```diff
@@ -26,22 +27,30 @@
  * This class contains the main activity that is started you run the project.
  */
 public class MainActivity extends AndroidHarness {
+
     private Main application;
     private SensorManager mSensorManager;
     private AccelerometerSensor accelerometerSensor;
     private PositionHolder posHolder;
     private AttackMessenger atkMessenger;
+    private HealthMessageHandler healthListener;
+    private HitMissMessageHandler hitMissListener;
     private HeartsUpdateFunctions huFunctions;
     private MakeButtonFunctions mbFunctions;
     private PlatformPosition position;
     private ClientHub clientHub = ClientHub.getInstance();
     private StartBugEventMessageListener startBugEventListener;
     private ClientWrapper client;
+    private ImageView img1, img2, img3;
```

Comment:
it might be easier to make an array/arraylist of imageViews. This would also get rid of the two switch statements